app/src/components/market/sections/market_buy/scalar_market_buy.tsx

```diff
@@ -121,7 +121,7 @@ export const ScalarMarketBuy = (props: Props) => {
     context,
   )
   const collateralBalance = maybeCollateralBalance || Zero
-  const walletBalance = formatNumber(formatBigNumber(collateralBalance, collateral.decimals, 5), 5)
+  const walletBalance = formatNumber(formatBigNumber(collateralBalance, collateral.decimals), 5)
```

Comment:
Shouldn't be removing the third param from `formatBigNumber` here as it will return an improper value for `walletBalance`. Looks like this is done in other places too